index.html

```diff
@@ -1,26 +1,113 @@
-<!DOCTYPE html> 
+<!DOCTYPE html>
 <html lang="en">
-  
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="device-width, initial-scale=1.0"> 
-    <link rel="stylesheet" href="styles.css"> 
-   </head>
-  
-  <body>
-    <header>Glazed Donuts</header> 
-    
-    <main> </main>
-    
-    
-    
-    
-    
-    
-    <!--https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fglazed%20donut01.jpg?v=1602793076372 --> 
-    <!--https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fglazed%20donut02.jpg?v=1602793077492 --> 
-    
-    
-  </body>
-  
-</html>
\ No newline at end of file
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+  <img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fglazed%20donut01.jpg?v=1602793076372" alt="">
+
+  <img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fspiceracklogo.png?v=1603324422595" alt="">
+
+  <h1>glazed Donuts</h1>
+
+  <ul>
+    <li><img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fcooktime.png?v=1603325721633" alt=""> 3
+      hr 30 min</li>
+    <li><img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fserves.png?v=1603325721686" alt="">14
+      donuts</li>
+    <li><img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fingredientss.png?v=1603325721633"
+        alt="">Click for ingredients</li>
+    <li><img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fcomments.png?v=1603325721633"
+        alt="">Comments</li>
+  </ul>
+
+  <p>Glazed donuts. Everybody loves &apos;em. Even when seemingly everyone is paleo this and gluten-free that, you
+    rarely see someone refuse a glazed donut. Especially when they&apos;re homemade, hot from the oil, and taste just
+    like the old-fashioned kind you remember from childhood. This particular recipe turns out donuts that aren&apos;t
+    quite as airy as Krispy Kreme donuts, but we don&apos;t think that&apos;s a bad thing. The donuts are less sweet
+    than most, which means you can drown them in glaze and they still won&apos;t seem cloying. Talk about our kinda
+    donut. Seriously. I can&apos;t stop eating these donuts. This note may have been typed impatiently and one-handedly
+    while attempting to lick drippy donut glaze from fingers and wrist before it plummets to certain disaster onto my
+    laptop.</p>
+
+  <img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Flike.png?v=1603327411736" alt="">
+
+  <img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Freadon.png?v=1603325721697" alt=""> 
+
+<section> 
+  <ul> For the Donuts 
```

Comment:
The only element that can be inside an unordered or ordered list is a `li` element. So, here, I'd take your text out of the list and instead make it a heading.
```suggestion
  <h3>For the Donuts</h3>
  <ul>
```